cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +91,77 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func CheckProbePeriodSeconds(elem *v1.Probe, cut *provider.Container, s string) bool {
```

Comment:
I feel like you could remove `s` and determine the type of the probe with `reflect` to not have to pass in the string of what type of probe we're looking at.

Reply:
I was trying to find field that give this data(if is StartupProbe/ LivenessProbe / ReadinessProbe)  but i didn't find.